code/ui/tttpanel/TTTPanel.cs

```diff
@@ -14,13 +14,13 @@ public bool IsShowing
                 SetClass("hide", !_isShowing);
             }
         }
-        private bool _isShowing = false;
+        private bool _isShowing = true;
```

Comment:
I'm assuming you checked, but we have no elements that are now showing that used to not show, right? I think we should be covered.